activity_hub.php

```diff
@@ -95,6 +95,23 @@
     $n_projects_in_state_[$project_state] = $count;
 }
 
+// SR projects have an additional qualifier beyond just the state
+// -- It can't be added to $n_projects_in_state_ because that
+// messes with the totals for the projects in PP, so create another
+// array just for SR.
+$sql = "
+    SELECT state, COUNT(*)
+    FROM projects
+    WHERE state = 'proj_post_first_checked_out' AND
```

Comment:
should we reference https://github.com/DistributedProofreaders/dproofreaders/blob/e29261c82e30c88fe38e7db9740dff8155c02f6a/pinc/project_states.inc#L282 instead just in case it ever changes or someone wonders where these states come from?

Reply:
Finally done